vtex-apps/checkout-components
# CHK-425: Update group options borders and caret icon

#### What problem is this solving?

Update the border of the `GroupOption` component (removed the top border, and remove the divider at the bottom in favor of a bottom border in the button element), and changed the caret component from the styleguide version to a custom one.

#### How should this be manually tested?

[Workspace](https://lucas--checkoutio.myvtex.com/cart/add?sku=289).

You can see the updated component in the shipping calculator and in Checkout.

#### Checklist/Reminders


#### Screenshots or example usage

Before

![image](https://user-images.githubusercontent.com/10223856/97706281-2e590500-1a94-11eb-925b-ca59aef3ecfc.png)

After

![image](https://user-images.githubusercontent.com/10223856/97706345-44ff5c00-1a94-11eb-8ee7-4306c88852cc.png)

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes
